### DISCUSSION TITLE
New ambiguous highlighting

### DISCUSSION THREAD

**Discussion by Mazan-ka** [dbeaver/dbeaver#22262] [Q&A]
Hello. I have installed DBeaver 23.3.1, and my SQL code become multicolor, how can I turn off this highlighting? 

<img width="260" alt="image" src="https://github.com/dbeaver/dbeaver/assets/80702083/62e10cc6-3190-4b7f-b3b4-f5473a3f285c">

<img width="145" alt="image" src="https://github.com/dbeaver/dbeaver/assets/80702083/254e2f47-f37e-4fc4-9b41-94e3f80c9d1b">

**Comment by LonwoLonwo** [comment]
Hello @Mazan-ka 

You can do it in your Preferences -> Editors -> SQL Editor -> Code Editor -> Query Analysis

![2023-12-27 13_08_00-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/3ba0c916-74af-4255-81a2-bb6752126611)

**Reply by Mazan-ka** [reply]
Thanks! It helped

**Comment by E1izabeth** [comment]
What database do you use?

**Reply by Mazan-ka** [reply]
GreenPlum
